urllib3/connection.py

```diff
@@ -162,6 +162,37 @@ def connect(self):
         conn = self._new_conn()
         self._prepare_conn(conn)
 
+    def request_chunked(self, method, url, body=None, headers={}):
+        """
+        Alternative to the common request method, which sends the
+        body with chunked encoding and not as one block
+        """
+        header_names = set(k.lower() for k in headers)
+        self.putrequest(method, url, skip_accept_encoding=True)
```

Comment:
Though, I should mimic the original behavoiur better and check for an already set accept-encoding header bevor skipping...